README.md

```diff
@@ -56,17 +56,19 @@ Then at the top of your script require the autoloader:
 require 'vendor/autoload.php';
 ```
 
-## Alternative: Install from zip
+#### Alternative: Install package from zip
 
-If you are not using Composer, simply download and install the **[latest packaged release of the library as a zip](https://sendgrid-open-source.s3.amazonaws.com/smtpapi-php/smtpapi-php.zip)**.
+If you are not using Composer, simply download and install the **[latest packaged release of the library as a zip](https://github.com/sendgrid/smtpapi-php/releases/download/0.6.1/smtpapi-php.zip)**.
+
+[**⬇︎ Download Packaged Library ⬇︎**](https://github.com/sendgrid/smtpapi-php/releases/download/0.6.1/smtpapi-php.zip)
```

Comment:
Need to add this to UpdateSendGridRubySMTPAPIFiles: https://code.hq.twilio.com/twilio/librarian/pull/178/files#diff-0b1d22901bca0c380969a6ee6b62cdd0 (sub of version number)

Reply:
Same as above.